pkg/cli/scan.go

```diff
@@ -18,52 +18,15 @@ package cli
 
 import (
 	"fmt"
-	"os"
 	"strings"
 
 	iacProvider "github.com/accurics/terrascan/pkg/iac-providers"
 	"github.com/accurics/terrascan/pkg/policy"
-	"github.com/mattn/go-isatty"
 	"github.com/spf13/cobra"
 	"go.uber.org/zap"
 )
 
-var (
-	// PolicyPath Policy path directory
-	PolicyPath []string
-
-	// PolicyType Cloud type (aws, azure, gcp, github)
-	PolicyType []string
-
-	// IacType IaC type (terraform)
-	IacType string
-
-	// IacVersion IaC version (for terraform:v12)
-	IacVersion string
-
-	// IacFilePath Path to a single IaC file
-	IacFilePath string
-
-	// IacDirPath Path to a directory containing one or more IaC files
-	IacDirPath string
-
-	// RemoteType indicates the type of remote backend. Supported backends are
-	// git s3, gcs, http.
-	RemoteType string
-
-	// RemoteURL points to the remote Iac repository on git, s3, gcs, http
-	RemoteURL string
-
-	// ConfigOnly will output resource config (should only be used for debugging purposes)
-	ConfigOnly bool
-
-	// UseColors indicates whether to use color output
-	UseColors bool
-	useColors string // used for flag processing
-
-	// Verbose indicates whether to display all fields in default human readlbe output
-	Verbose bool
-)
+var scanOptions = new(ScanOptions)
```

Comment:
Also, can have a function like `NewScanOptions()` which returns a new empty `*ScanOptions` struct?

Reply:
I did not add any constructor function for ScanOptions struct because we are not doing any initialisation based on parameters.